### ISSUE TITLE
Hi, the latest update seems can't work with AUX addon

### ISSUE THREAD

**Issue by hbltth** [shagu/pfUI#1371]
I update just now and the AUX addon doesn't work, an error 'attempt to index 'table'(a nil value)'.

**Comment by shagu** [comment]
I use aux and don't have that error. Also there's no code in pfUI that is modifying aux in any way and there haven't been made any recent changes that could influence aux at all. If you have aux errors, please open a ticket on the aux addon issue tracker, not here. Also I suggest to tell the full error message if you do.

**Comment by hbltth** [comment]
When I disable PfUI then  AUX can work again now.
ERROR: Interface\AddOns\aux-addon\tabs\search\core.lua:29: attempt to index field `table' (a nil value).

**Comment by shagu** [comment]
1. Which aux addon are you using? 
2. Any other errors if you scroll up? 
3. When does it happen / how to reproduce it? 
4. Have you tried clearing your WDB Folder already? 
5. Which pfUI version did you upgrade from before? 
6. Does it happen on another character aswell? 
7. Have you tried disabling **all** other addons except pfUI and aux? 
8. Are you sure that pfUI is not just the messenger that just displays the error that is otherwise hidden by another addon?

**Comment by shagu** [comment]
@gwetchen, as it is your aux-addon that is mentioned in the Turtle-WoW wiki: Do you see any way how those two lines can have anything to do with pfUI? Thanks in advance.

https://github.com/gwetchen/aux-addon/blob/master/tabs/search/core.lua#L29
https://github.com/gwetchen/aux-addon/blob/master/tabs/search/results.lua#L430

**Comment by shagu** [comment]
I'm now using the latest version of @gwetchen's version of `aux-addon` and still can't reproduce it.
Waiting for more input.

**Comment by hbltth** [comment]
2. I scrolled up and all error messages are the same.
3. Before the latest PfUI update the 2 addons works well. And there is a '!libs' folder in my addon folder, does it matter with this?
4. I have deleted WDB.
6. Not only this character but other client happens too.
7. I disabled other addons except PfUI and AUX, and no errors.

**Comment by hbltth** [comment]
> * I scrolled up and all error messages are the same.
> * Before the latest PfUI update the 2 addons works well. And there is a '!libs' folder in my addon folder, does it matter with this?
> * I have deleted WDB.
> * Not only this character but other client happens too.
> * I disabled other addons except PfUI and AUX, and no errors. Maybe I should find out which addon is the problem.

**Comment by shagu** [comment]
> I disabled other addons except PfUI and AUX, and no errors. Maybe I should find out which addon is the problem.

You could now enable them back one by one and see which one did it.

I mean, If you tell me what version of pfUI you had before, I can just send you that version again. But I still highly doubt it has anything to do with it. As I just can't see anything in the recent changes that could break aux in that way.

**Comment by shagu** [comment]
Just in case: Here's the link to the previous version of pfUI:
https://github.com/shagu/pfUI/archive/a9cdbbda539b6688b3f7cc566563096da5477235.zip

If it's really the pfUI update and not an update of another addon that broke it, the error should be gone after downgrading pfUI to that version. Please let me know if you found out more details.

**Comment by hbltth** [comment]
> Just in case: Here's the link to the previous version of pfUI: https://github.com/shagu/pfUI/archive/a9cdbbda539b6688b3f7cc566563096da5477235.zip
> 
> If it's really the pfUI update and not an update of another addon that broke it, the error should be gone after downgrading pfUI to that version. Please let me know if you found out more details.

Ok, thanks ,I will try.

**Comment by hbltth** [comment]
> Just in case: Here's the link to the previous version of pfUI: https://github.com/shagu/pfUI/archive/a9cdbbda539b6688b3f7cc566563096da5477235.zip
> 
> If it's really the pfUI update and not an update of another addon that broke it, the error should be gone after downgrading pfUI to that version. Please let me know if you found out more details.

I used the old version PfUI and no error happened. But when I deleted the old version folder and pasted the latest version folder, errors happend again.

**Comment by shagu** [comment]
Can you please switch to the latest version again, go into the pfUI settings and disable the "pixelperfect" module in the components section? :eyes:

**Comment by hbltth** [comment]
> Can you please switch to the latest version again, go into the pfUI settings and disable the "pixelperfect" module in the components section? 👀

Yeah, I disabled it and no error again.

**Comment by shagu** [comment]
- Your Setup + New pfUI: Error
- Your Setup + Old pfUI: No Errors
- No Addons, only pfUI + Aux: No Errors

So to sum things up:
The recent update makes one of your addons misbehave, which then in return breaks aux.

I'm watching the recent change again and again.. I'm really interested how any addon could break by that.
Please let me know if you found the one that can't handle the latest pfUI.

**Comment by hbltth** [comment]
> * Your Setup + New pfUI: Error
> * Your Setup + Old pfUI: No Errors
> * No Addons, only pfUI + Aux: No Errors
> 
> So to sum things up: The recent update makes one of your addons misbehave, which then in return breaks aux.
> 
> I'm watching the recent change again and again.. I'm really interested how any addon could break by that. Please let me know if you found the one that can't handle the latest pfUI and kills aux.

Sure.

**Comment by shagu** [comment]
I pushed a new update, that *could* solve it? I'm just guessing into the dark here.
https://github.com/shagu/pfUI/commit/3975ffb334e9b88e7e3ba7426e6df8a0954cf49e

But out of curiousity, I'd be still very interested in which of the addons went nuts

**Comment by hbltth** [comment]
> I pushed a new update, that _could_ solve it? I'm just guessing into the dark here. [3975ffb](https://github.com/shagu/pfUI/commit/3975ffb334e9b88e7e3ba7426e6df8a0954cf49e)
> 
> But out of curiousity, I'd be still very interested in which of the addons went nuts

Yeah the new update solves the problem and no error, and no worries I will try to find out which addon :).

**Comment by hbltth** [comment]
> I pushed a new update, that _could_ solve it? I'm just guessing into the dark here. [3975ffb](https://github.com/shagu/pfUI/commit/3975ffb334e9b88e7e3ba7426e6df8a0954cf49e)
> 
> But out of curiousity, I'd be still very interested in which of the addons went nuts

I tried, and found that disabled AutoDB2 or PfQuest, there is no error.
PfUI + AUX + PfQuest + AutoDB2 = Error.
PfUI + AUX + PfQuest = No Error.
PfUI + AUX + AutoDB2 = No Error.
